src/Magnum/OvrIntegration/Hmd.cpp

```diff
@@ -160,16 +160,39 @@ bool Hmd::isDebugHmd() const {
     return (_flags & HmdStatusFlag::Debug) != HmdStatusFlags{};
 }
 
-std::array<DualQuaternion, 2> Hmd::eyePoses() {
-    return std::array<DualQuaternion, 2>{{DualQuaternion(_ovrPoses[0]), DualQuaternion(_ovrPoses[1])}};
-}
-
 void Hmd::setPerformanceHudMode(const PerformanceHudMode mode) const {
-    ovr_SetInt(_hmd, OVR_PERF_HUD_MODE, Int(mode));
+    ovr_SetInt(_session, OVR_PERF_HUD_MODE, Int(mode));
 }
 
 void Hmd::setDebugHudStereoMode(const DebugHudStereoMode mode) const {
-    ovr_SetInt(_hmd, OVR_DEBUG_HUD_STEREO_MODE, Int(mode));
+    ovr_SetInt(_session, OVR_DEBUG_HUD_STEREO_MODE, Int(mode));
+}
+
+Hmd::EyeCup Hmd::eyeCup() const {
+    char value[16];
+    ovr_GetString(_session, OVR_KEY_EYE_CUP, "A");
+    if(strcmp(value, "A")) {
```

Comment:
`std::strcmp`, please. I guess there is a reason why they are using string and not enum? For example, wouldn't this be vendor-dependent or something?